apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -110,6 +113,18 @@ export class WidgetsController {
     return await this.markMessageAsSeenUsecase.execute(command);
   }
 
+  @UseGuards(AuthGuard('subscriberJwt'))
+  @Post('/messages/seen')
```

Comment:
Could you please add an `e2e` test for the API on this endpoint?

Reply:
Sure, I forgot to add. I will add